### PR TITLE
update full project to new dev standards part 1

### PR DESCRIPTION
**Pull request by jeffkala**

### CODE REVIEW

File: docs/user/lib_getting_started.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+The first way is to use the `ntc_device` object. Just pass in all required parameters to the object to initialize your device.  Here we are showing the import, but renaming the object to `NTC`.
+
+```
+>>> from pyntc import ntc_device as NTC
+>>>
+```
+
+Like many libraries, we need to pass in the host/IP and credentials.  Because this is a multi-vendor/API library, we also use the `device_type` parameter to identify which device we are building an instance of.
+
+pyntc currently supports four device types:
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:14]:
```suggestion
pyntc currently supports seven device types:
```

File: docs/user/lib_getting_started.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+* cisco_asa_ssh
+* cisco_ios_ssh
+* cisco_nxos_nxapi
+* arista_eos_eapi
+* juniper_junos_netconf
+* f5_tmos_icontrol
+
+The example below shows how to build a device object when working with a Cisco IOS router.
+
+```
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:25]:
```suggestion
```python
```

File: docs/user/lib_getting_started.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+>>> # CREATE DEVICE OBJECT FOR AN IOS DEVICE
+>>>
+>>> csr1 = NTC(host='csr1', username='ntc', password='ntc123', device_type='cisco_ios_ssh')
+>>>
+```
+
+And here is an object for a Cisco Nexus device:
+
+```
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:34]:
```suggestion
```python
```

File: docs/user/lib_getting_started.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+The second way to get started with pyntc is to use the pyntc configuration file.  This was modeled after Arista's `.eapi.conf` file.  Our file is called `.ntc.conf`
+
+This simplifies creating device objects since you no longer need to specify credentials and other device specific parameters when you build the device object.  Instead, they are stored in the conf file.
+
+
+# pyntc Configration File
+
+- filename:  `.ntc.conf`
+- Priority of locating the conf file:
+  - `filename` param in `ntc_device_by_name`
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:50]: i think these need to be 4 spaces.

File: docs/user/lib_getting_started.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+  - Environment Variable aka `PYNTC_CONF`
+  - Home directory `.ntc.conf`
+- Specify device_type and a name
+- host is not required if the name is the device's FQDN
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:54]: `host` or Host

File: docs/user/lib_getting_started.md [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+[juniper_junos_netconf:vmx1]
+host: 176.126.88.99
+username: ntc
+password: ntc123
+
+```
+
+We can now build device objects just by referencing the name of the device from the conf file.
+
+```
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:81]:
```suggestion
```python
```

File: docs/user/lib_getting_started.md [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+
+Note: the only method and property not supported on all devices is `install_os`.  It is not supported on Juniper Junos devices.
+
+### Gathering Facts
+
+- Use `facts` device property
+
+On a Nexus device:
+
+```
```
Comment (itdependsnetworks) [inline docs/user/lib_getting_started.md:102]:
```suggestion
```python
```

Comment (itdependsnetworks) [reply docs/user/lib_getting_started.md:102]: there is a bunch of these :) can you knock them out?

Comment (jeffkala) [reply docs/user/lib_getting_started.md:102]: thanks, just copy/pasted from old readme, will get them all updated.

**Review by jvanderaa — APPROVED** [review]

**Pull request merged**